lib/usbd/usbd.c

```diff
@@ -224,6 +234,8 @@ bool usbd_is_vbus(usbd_device *dev){
 }
 
 void usbd_enable(usbd_device *dev, bool onoff){
+	if (!onoff)
+		usbd_disconnect(dev, false);
 	if (dev->backend->power_control)
 		dev->backend->power_control(dev, (onoff)? usbd_paActivate : usbd_paShutdown );
 }
```

Comment:
When `usbd_enable(dev, false)` is called, it disconnect and shutdown. but when `usbd_enable(dev, true)` is called, it only power-up (it remove the disconnect condition).